src/components/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Link} from "react-router-dom";
+
+class Login extends React.Component {
+  constructor(props) {
+    super();
+    console.log(props);
+    this.state = {
+      username: "",
+      password: "",
+    };
+  }
+
+  //*ignore using below as a template for now until i get api*
+  //  SubmitButton = (e) => {
+  //     e.preventDefault();
+  //     console.log(this.state)
+  //     axios
+  //       .post(`http://localhost:5000/api/login`, this.state)
+  //       .then((response) => {
+  //         console.log(response)
+  //         localStorage.setItem("token", response.data.payload);
+  //         this.props.history.push("/protected");
+  //         console.log(response);
+  //       })
+  //       .catch((err) => {
+  //         console.log(err);
+  //       });
+  //   };
```

Comment:
Do not leave commented code in your commits.
This is known as [zombie code](https://www.codelord.net/2011/10/28/fight-zombie-code) and I won't merge PRs that have zombie code.